WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingStreamsViewController.swift

```diff
@@ -155,6 +156,20 @@ public class NotificationSettingStreamsViewController : UITableViewController
         return sortedStreams![section]
     }
     
+    private func imageForStreamKind(streamKind: NotificationSettings.Stream.Kind) -> UIImage? {
+        let imageName : String
+        switch streamKind {
+        case .Email:
+            imageName = "notifications-email"
+        case .Timeline:
+            imageName = "notifications-bell"
+        case .Device:
+            imageName = "notifications-phone"
+        }
```

Comment:
In Swift, if you don't add a case for every single (enum case), the compiler will kick your head off. No need to shortcut, in this land.

Thanks for the paranoia!! :dancer: